app/main.py

```diff
@@ -126,6 +126,10 @@ async def start_up():
 def periodic_execution():
     global FILTER_GENERATED
     FILTER_GENERATED = fg.generate_filter_dictionary(SUBMISSION)
+    retry = 0
+    while not FILTER_GENERATED:
+        time.sleep(retry + 1)
```

Comment:
FILTER_GENERATED = fg.generate_filter_dictionary(SUBMISSION)
Should be called in the loop otherwise the thread will be on sleep till the next time the periodic_execution will be executed again?
I take a look at the generate_filter_dictionary method and it seems to return true only, perhaps it needs some exception handling in there.